app/src/main/java/com/github/se/icebreakrr/data/AppDataStore.kt

```diff
@@ -127,4 +137,14 @@ open class AppDataStore(private val dataStore: DataStore<Preferences>) {
   private suspend fun <T> removePreference(key: Preferences.Key<T>) {
     dataStore.edit { preferences -> preferences.remove(key) }
   }
+
+  suspend fun saveNotificationTime(uid: String, timestamp: Long) {
+    putPreference(longPreferencesKey(NOTIFICATION_TIME_PREFIX + uid), timestamp)
+  }
+
+  suspend fun clearNotificationTime(uid: String) {
+    dataStore.edit { preferences ->
+      preferences.remove(longPreferencesKey(NOTIFICATION_TIME_PREFIX + uid))
+    }
+  }
```

Comment:
Lack of documentation for those two functions, it isn't clear enough what they are doing